### PR TITLE
Update library_ccall.js

### PR DESCRIPTION
**Pull request by Ishant-Modi**

### CODE REVIEW

File: src/library_ccall.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ mergeInto(LibraryManager.library, {
 #endif
       'string': (str) => {
         var ret = 0;
+         if (typeof str !== 'string') {
+    throw new TypeError('Argument must be a string');
+  }
```
Comment (sbc100) [inline src/library_ccall.js:36]: How about:

```
#if ASSERTIONS
  assert(typeof str == 'string');
#endif
```

And maybe at a test for this?  (or modify one of the existing ccall tests).

Comment (Ishant-Modi) [reply src/library_ccall.js:36]: That could work but showing the user the error they have made while giving an input would be more worthwhile and would guide the user in providing another prompt much more successfully.

Comment (sbc100) [reply src/library_ccall.js:36]: Sure, you can do `assert(typeof str == 'string', 'Argument must be a string');` if you want to include a message.

Comment (Ishant-Modi) [reply src/library_ccall.js:36]: Great! let me just implement it then.

**Pull request closed without merging**